src/pages/nutrition/productCard.jsx

```diff
@@ -37,6 +39,14 @@ export default function ProductNutriments({ setNutriments, nutriments }) {
       setIndex(0);
       setPage((prevPage) => ++prevPage);
     }
+    const resArr = nutriments.filter(
+      (nutr) => nutr.display && +nutr.value && +nutr.value > 0
+    );
```

Comment:
What does `resArr` mean?

A value can be 0. For example knowing that some coca has 0 sugar is an information
```suggestion
    const resArr = nutriments.filter(
      (nutr) => nutr.display && nutr.value !== null && nutr.value >= 0
    );
```